test/test_SW_SoilWater.cc

```diff
@@ -27,20 +27,17 @@ namespace{
   // Test the 'SW_SoilWater' function 'SW_VWCBulkRes'
   TEST(SWSoilWaterTest, SWVWCBulkRes){
     //declare mock INPUTS
-    RealD fractionGravel = .1;
-    RealD clay = .7;
-    RealD sand = .2;
-    RealD porosity = 1;
+    RealD fractionGravel = .1, clay = .7, sand = .2, porosity = 1;
 
     // test clay > .6
     RealD res = SW_VWCBulkRes(fractionGravel, sand, clay, porosity);
     EXPECT_DOUBLE_EQ(res, SW_MISSING);
```

Comment:
Can you put a note, inline with this test (and others), that you expect it be missing because of XYZ. Explicitly state why you expect your test to pass, fail, etc.